client/src/components/DataFiles/DataFilesStatus/DataFilesStatus.js

```diff
@@ -42,16 +42,31 @@ const OPERATION_MAP = {
           .slice(0, -1)
           .join('/');
         const projectName = findProjectTitle(projectList, response.systemId);
+
+        const srcSystem =
+          response.source.split('/')[0] === 'https:'
+            ? response.source.split('/')[7]
+            : response.source.split('/')[2];
+        const destSystem = response.systemId;
+
+        let op = mappedOp;
+        let dest;
+
+        if (srcSystem !== destSystem && mappedOp === 'copied') {
```

Comment:
We're already in the `copy` block, so I think `mappedOp === 'copied'` is redundant
```suggestion
        if (srcSystem !== destSystem) {
```